src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -214,8 +214,8 @@
 
   &.visible::after {
     position: absolute;
-    height: auto;
-    bottom: auto;
+    height: 31px;
```

Comment:
this seems very specific. is there another way to achieve what you are looking for ?

i can help.

Reply:
i was able to figure out a solution, check out the updates